packages/sdk-react/src/components/counterparts/CounterpartDetails/CounterpartBankForm/CounterpartBankForm.test.tsx

```diff
@@ -37,7 +39,9 @@ describe('CounterpartBankForm', () => {
       </MoniteStyleProvider>
     );
 
-    await waitUntilTableIsLoaded();
+    await waitFor(() => checkCounterpartQueriesLoaded(testQueryClient), {
```

Comment:
Hello. I do have 2 questions:
1. Why `waitUntilTableIsLoaded` is unstable?
2. Default timeout is 1 second. Increasing timeout to 5 or even 10 seconds looks like hack and if everything has been written correct timeout to 1 second should be more than enough

Reply:
> For example, there may be 2 spinners appearing sequentially on screen (as in Super Components, for example).

I dunno, for me it's a mistake if we have 2 or more spinners sequentially. I just curious why in other tests `waitUntilTableIsLoaded` works but in `CounterpartBankForm` isn't.

From my point of view there is some mistake inside `CounterpartBankForm` and it's not related to `waitUntilTableIsLoaded` otherwise all our tests will fail but right now only this one fails. So I'd like to check what have been changed inside the logic why it's so error-prone

Reply:
In my opinion, it is not necessary to use `waitUntilTableIsLoaded `.

We can use this code instead:
```ts
const submitBtn = await waitFor(
  () =>
    screen.getByRole('button', {
      name: t`Add bank account`,
    })
);